docs/send-data/collect-from-other-data-sources/azure-blob-storage/index.md

```diff
@@ -14,6 +14,21 @@ This solution is for newly created blobs only (not for existing blobs). 
 
 For step-by-step instructions for configuring the Azure-Sumo pipeline, see [Collect Logs from Azure Blob Storage](collect-logs-azure-blob-storage.md).
 
+Watch this tutorial for an overview of collecting logs from Azure Blob Storage.
+
+<Iframe url="https://www.youtube.com/embed/_-N3QGxrn9k?rel=0"
+        width="854px"
+        height="480px"
+        id="myId"
+        className="video-container"
+        display="initial"
+        position="relative"
+        allow="accelerometer; autoplay=1; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+        />
+
+import Iframe from 'react-iframe';
```

Comment:
Can we move this import Iframe command to top of the page.

Reply:
Done!